src/routes/SheltersPege/SheltersPege.js

```diff
@@ -0,0 +1,187 @@
+import React, { Component } from 'react';
+import { Section, FormGroup } from '../../components/Utils/Utils';
+import './SheltersPege.css';
+import SheltersService from '../../services/sheltersService';
+import PetCard from '../../components/PetCard/PetCard';
+
+export default class SheltersPege extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      shelterID: null,
+      shelter: null,
+      pets: null,
+      typeOfPets: "Any",
+      age: "Any",
+      availability: "Any"
+    };
+  }
+
+  componentDidMount() {
+    console.log("Getting shelter data...")
+    const path = window.location.pathname;
+    const shelterID = path.substring(path.lastIndexOf('/') + 1)
+    SheltersService.getShelter(shelterID)
+      .then(shelter => this.setState({ shelter: shelter }))
+      .catch(error => console.log(error));
+
+    console.log("Getting pets data...")
+    SheltersService.getPetsByShelter(shelterID)
+      .then(pets => this.setState({ pets: pets }))
+      .catch(error => console.log(error));
+  }
+
+  renderShelter(shelter) {
+    console.log("Calling renderShelter...")
+    return (
+      <div>
+        <h2>{shelter[0].ShelterName}</h2>
+        <hr />
+        <div>
+          <table className='shelterTable'>
```

Comment:
I prefer not to have lines between rows. That's why I used non-bootstrap class. I can see that using bootstrap class is handy if we want to work on the mobile page.
![image](https://user-images.githubusercontent.com/71732016/153671795-316f4b0c-cfe5-4b5d-b657-334397b0242c.png)

Reply:
I agree that it looks good without the lines. If it's more responsive, we could always use the bootstrap class and then add custom css styling like `.table { border-width: 0 }`. Otherwise I think it's good as-is

Reply:
I just tested the table without bootstrap and it's responsive enough (because the table data is so small). Thanks!